http-server/src/main/java/io/micronaut/http/server/HttpServerConfiguration.java

```diff
@@ -478,6 +492,13 @@ public void setDualProtocol(boolean dualProtocol) {
         this.dualProtocol = dualProtocol;
     }
 
+    /**
+     * @param httpToHttpsRedirect the redirect HTTP to HTTPS configuration
```

Comment:
Perhaps expand on this to denote that it should only be used with dual protocol

Reply:
Done